miden-tx/src/tests/kernel_tests/test_prologue.rs

```diff
@@ -357,16 +357,15 @@ fn input_notes_memory_assertions(
 #[test]
 pub fn test_prologue_create_account() {
     let (account, seed) = AccountBuilder::new(ChaCha20Rng::from_entropy())
-        .account_type(miden_objects::accounts::AccountType::RegularAccountUpdatableCode)
-        .build(TransactionKernel::assembler())
+        .build(TransactionKernel::assembler_testing().clone())
```

Comment:
Similar to the above comment: do we actually need to use the testing assembler here? I think regular assembler may work fine.

Reply:
I think it's fine to keep as is.